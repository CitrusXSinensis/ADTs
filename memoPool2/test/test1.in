ra10 5 4 2x
pa
aa 45
pa
ca
fa 3
pa
fa 6
pa
ca
fa 200

