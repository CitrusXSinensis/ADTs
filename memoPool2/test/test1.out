10 5 4 2
10 5 4 2 45
5/8
Chunk: 10
5 4 2 45
Chunk: 45
5 4 2
3/8
Nullptr
